config/initializers/redis.rb

```diff
@@ -9,14 +9,16 @@
   ENV['REDIS_URL'] = "redis://#{password.blank? ? '' : ":#{password}@"}#{host}:#{port}/#{db}"
 end
 
-Redis.current = Redis.new(
+namespace = ENV.fetch('REDIS_NAMESPACE') { '' }
```

Comment:
i don't see why not because all it is is a string that's prepended to keys when doing redis stuff, so that should be a no-op

however for the sake of full disclosure i did not test that because i'm a monster who still doesn't have a dev environment and switching to an empty namespace would have resulted in everyone seeing empty timelines on my instance i used for testing

i'm not sure i understand the concern about purging subscriptions. can you tell me more?